Nodejs/Product/Nodejs/Intellisense/VsProjectAnalyzer.cs

```diff
@@ -1075,41 +1075,6 @@ CollectingErrorSink errorSink
         ) {
             // There are some scenarios during ES6 Mode where a textview is still opened using NodeLS.
             // In these cases, we do not properly parse ES6 syntax, and therefore do not want to display errors. 
-            if (NodejsPackage.Instance.IntellisenseOptionsPage.AnalysisLevel == AnalysisLevel.Preview) {
-                return;
-            }
```

Comment:
Do you mean remove the entire If block? This code is unreachable after this change if we keep the current logic.